package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@legumeinfo/graphql-server",
-  "version": "1.3.1",
+  "version": "1.4.0",
```

Comment:
You need to run `npm install` after bumping the version in `package.json` so that it gets updated in `package-lock.json` as well.